servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestMisc.java

```diff
@@ -51,8 +50,10 @@ public void checkSpecialCharacterRoundTrip() throws BaseNessieClientServerExcept
         .commitMeta(CommitMeta.fromMessage("commit 1"))
         .commit();
 
-    assertThat(getApi().getContent().key(k).refName(branch.getName()).get()).containsEntry(k, ta);
-    assertEquals(ta, getApi().getContent().key(k).refName(branch.getName()).get().get(k));
+    assertThat(getApi().getContent().key(k).refName(branch.getName()).get())
+        .containsKey(k)
+        .hasEntrySatisfying(
+            k, c -> assertThat(c).isEqualTo(IcebergTable.builder().from(ta).id(c.getId()).build()));
```

Comment:
nit: rename `k` to about confusion with `k` in the outer scope.

Reply:
(same)